### DISCUSSION TITLE
服务端启动好像报错了

### DISCUSSION THREAD

**Discussion by Mronezc** [zdz/ServerStatus-Rust#109] [Q&A] [answered]
按照快速体验运行起来了，但是我想修改一下配置文件管理员密码尝试运行了以下命令
./stat_server -h
./stat_server -c config.toml
RUST_BACKTRACE=1 RUST_LOG=trace ./stat_server -c config.toml

均会出现
thread 'main' panicked at 'error binding to 0.0.0.0:8080: error creating server listener: Address in use (os error 98)', /cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.14.23/src/server/server.rs:79:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
不知道是不是自己搭错了

**Comment by zdz** [comment, marked as answer]
8080端口已经被其它程序占用了，用命令看看哪个程序占用了
```bash
ss -nlapt | grep 8080
```

**Reply by Mronezc** [reply]
自己太笨了 快速体验，服务管理脚本，服务端运行客户端说明这几个绕蒙了感觉大佬的脚本好像比别的多一个步骤，中午弄得时候确实端口被占用了换一个就好了，不过我已经换别的了，有时间在仔细琢磨琢磨你的
